### ISSUE TITLE
Hardcode python interpreter name still an issue

### ISSUE THREAD

**Issue by jdwx** [mapbox/node-sqlite3#1044]
Hello,

The issue described in issue #967 appears to still be occurring.  (And I am reasonably sure I am the "manager" of the managed environment referenced there. :-) )

We're trying to build sqlite3@4.0.2 globally for that environment, and no matter what config steps are taken, including all the ones @worc originally tried plus setting the systemwide npmrc file, the sqlite3 build still has an action hardcoded to call "python."

In both master branch and on the v4.0.2 tag, this is in deps/sqlite3.gyp at line 58:

'action': ['python','./extract.py','./sqlite-autoconf-<@(sqlite_version).tar.gz','<(SHARED_INTERMEDIATE_DIR)']

Manually changing this to python2 seems to allow the build to succeed, but it seems like the proper fix would be to modify it to use the configured path to Python?

Thanks!

**Comment by foxxyz** [comment]
I think this has been resolved since dea2f3b784b. The way to specify the python interpreter to use is:

```
npm install sqlite3 --build-from-source --python=/path/to/python/binary
```
See: https://github.com/mapbox/node-sqlite3#source-install

**Comment by kewde** [comment]
Thank you @foxxyz.
